backend/src/api/auth/auth.api.ts

```diff
@@ -0,0 +1,23 @@
+import { Router } from 'express';
+import { login } from 'healthcare-shared/validation-schemas';
```

Comment:
```suggestion
import { login: loginSchema } from 'validation-schemas';
```
Don't forget about proxy-import